RDKShell/RDKShell.cpp

```diff
@@ -4555,39 +4562,27 @@ namespace WPEFramework {
         uint32_t RDKShell::getLastWakeupKeyWrapper(const JsonObject& parameters, JsonObject& response)
         {
             LOGINFOMETHOD();
-
-            if (nullptr == gSystemServiceConnection)
-            {
-                Utils::activatePlugin(SYSTEM_SERVICE_CALLSIGN);
-                std::cout << "Activated SystemService" << std::endl;
-            }
-
-            if (nullptr != gSystemServiceConnection)
+            
+            if (0 != mLastWakeupKeyTimestamp)
             {
                 JsonObject req, res;
+                gRdkShellMutex.lock();
```

Comment:
We may not need a lock here, as if thunder call blocked, it may block ui thread

Reply:
Removed.